src/components/CreateResetPassword/CreateResetPasswordControl.js

```diff
@@ -66,6 +69,12 @@ class CreateResetPasswordControl extends Component {
       }
     } = this.props;
 
+    const { isSuccessfulPasswordChange } = this.state;
+
+    if (isSuccessfulPasswordChange) {
+      return <SuccessScreen />;
```

Comment:
Please rename success screen PasswordSuccessfullyChanged. Do it here and in component file also.